sagemaker-python-sdk/tensorflow_distributed_mnist/tensorflow_batch_transform_mnist.ipynb

```diff
@@ -266,22 +234,87 @@
     "import json\n",
     "from urllib.parse import urlparse\n",
     "\n",
+    "import boto3\n",
+    "\n",
     "parsed_url = urlparse(transformer.output_path)\n",
     "bucket_name = parsed_url.netloc\n",
-    "file_key = '{}/{}.out'.format(parsed_url.path[1:], input_file_name)\n",
+    "prefix = parsed_url.path[1:]\n",
     "\n",
-    "output_obj = s3.Object(bucket_name, file_key)\n",
-    "output = output_obj.get()[\"Body\"].read()\n",
+    "s3 = boto3.resource('s3')\n",
     "\n",
-    "print('Prediction is {}'.format(json.loads(output)['outputs']['classes']['int64Val']))"
+    "predictions = []\n",
+    "for i in range(10):\n",
+    "    file_key = '{}/data-{}.csv.out'.format(prefix, i)\n",
+    "\n",
+    "    output_obj = s3.Object(bucket_name, file_key)\n",
+    "    output = output_obj.get()[\"Body\"].read().decode('utf-8')\n",
+    "\n",
+    "    predictions.extend(json.loads(output)['outputs']['classes']['int64Val'])"
```

Comment:
`NoSuchKey: An error occurred (NoSuchKey) when calling the GetObject operation: The specified key does not exist.` error.

Reply:
likely because the actual data isn't in the sample data bucket yet. I was planning on just replacing the old stuff, which is why I hadn't uploaded it yet.

although, do we need to have older versions of the notebook still work?

Reply:
If we can do it so that both versions work in parallel, I think that's best.

Reply:
moved to using a different folder within the bucket so that it works both ways. the data is in PDX and CMH - currently uploading to the other regions.